tests/lustre/typechecking/shouldFail/test_const_decls_tyalias.lus

```diff
@@ -0,0 +1,3 @@
+-- const r3_l = 1;
+const r3_u = 2;
+type r3 = subrange [r3_l, r3_u] of int;
```

Comment:
This test doesn’t fail. Fix it or move it temporally to a different directory.

Reply:
Type alias circularity check will be in the next PR. I will take this file out.